Project_Organization.md

```diff
@@ -51,16 +51,26 @@ to participate in decisions that require a Supermajority Vote.
 ### 3.1. Steering Committee
 
 The Steering Committee is the body that is responsible for governing the
-Project, coordinating Working Groups, and managing assets.
+Project, coordinating Working Groups, and managing assets. No more than two 
```

Comment:
Here is my suggestion for the phrasing to add: 
"To keep discussions productive and balanced, attendance is generally limited to two attendees per organization. The Steering Committee may grant an exception to this rule for a specific meeting and purpose only."